hangman_dinner.rb

```diff
@@ -0,0 +1,201 @@
+require "colorize"
+
+class Hangman
+    def initialize
+
+      @answer_bank = ["cupcake", "spaghetti", "cookie", "hamburger", "pizza", "sandwich", "crackers", "salad","cheese", "burrito"]
+      @guesses = 4
+      @num_correct = 0
+      @current_answer = @answer_bank[rand(@answer_bank.length)]
+      @current_answer_array = @current_answer.split(//)
+      @fillable_answer = []
+      @alphabet = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K","L", "M", "N", "O", "P", "Q", "R", "S",
+      "T", "U", "V", "W", "X", "Y", "Z"]
+      @already_guessed =[]
+      @current_answer_array.length.times do |x|
+        @fillable_answer[x]= " _ "
```

Comment:
There is a cute way of initializing an array with default values. The Array initializer (`.new`) can accept 2 arguments: How big to make the initial array, and a second argument of what to fill all slots with:
```
   @fillable_answer = Array.new(@current_answer_array.length, '_')
```